src/components/Editor/index.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { addModule, updateModule, deleteModule } from './actions.js';
+
+const styles = {
+    editor: {
+        width: '98%',
+        maxWidth: '750px',
+        margin: 'auto',
+        padding: '15px'
+    },
+    textarea: {
+        width: '100%',
+        maxWidth: '100%',
+        fontWeight: '300',
+        lineHeight: '2rem',
+        border: 'none',
+        outline: 'none',
+        resize: 'none',
+        background: '#f4f5f7',
+        padding: '1.5rem',
+        fontSize: '1.5rem'
+    },
+    text: {
+        width: '100%',
+        outline: 'none'
+    }
+};
+
+const Editor = ({ modules, actions }) => {
+    return (
```

Comment:
Since we aren't doing anything inside of this closure, we can just return the `( ... )` directly instead of the above.